Readme.md

```diff
@@ -53,8 +55,10 @@ The installer can now be found at `C:\Repos\VFSForGit\BuildOutput\GVFS.Installer
 
 ## Trying out VFS for Git
 
-* VFS for Git will work with any git service that supports the GVFS [protocol](Protocol.md). For example, you can create a repo in
-Azure DevOps (https://azure.microsoft.com/services/devops/), and push some contents to it. There are two constraints:
+* VFS for Git will work with any Git service that supports the
+  [GVFS protocol](Protocol.md). For example, you can create a repo in
+  [Azure DevOps](https://azure.microsoft.com/services/devops/), and push
+  some contents to it. There are two constraints:

```

Comment:
Do we really want to wrap the `.md`?

Reply:
...yes? It links to the page in the repo. Perhaps I don't understand your comment.